server/models/User.js

```diff
@@ -1,29 +1,51 @@
-/* This is a sample model file for the user schema. 
-This file will be used to create a user schema and export it to the server file. */
+const mongoose = require("mongoose");
+const bcrypt = require("bcrypt");
 
-import mongoose from 'mongoose'
+const userModel = mongoose.Schema({
+  id: { type: Number },
+  userName: { type: String },
+  password: { type: "String", required: true },
```

Comment:
Separate all the schema models in the different folder models and export all those models from there and then use them inside the model